File: geohelp-organizer-app/src/app/api/event-register-request.ts
export class EventRegisterRequest {

  public startTime: number = 0;
  public endTime: number = 0;
  public eventName: string = "";
  public locationLong: number[];
  public locationLat: number[];

  constructor(startTime: number, endTime: number, eventName: string, locationLong: number[], locationLat: number[]) {
    this.startTime = startTime;
    this.endTime = endTime;
    this.eventName = eventName;
    this.locationLong = locationLong;
    this.locationLat = locationLat;
  }

}

File: geohelp-backend/src/main/java/ch/geohelp/geohelpbackend/api/model/EventRegisterRequest.java
package ch.geohelp.geohelpbackend.api.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter

public class EventRegisterRequest {
    long startTime;
    long endTime;
    String eventName;
    double[] locationLong;
    double[] locationLat;
}
